cookbook/case_studies/ml_training/house_price_prediction/multiregion_house_price_predictor.py

```diff
@@ -1,17 +1,23 @@
 """
-
 Predicting House Price in Multiple Regions Using XGBoost and Dynamic Workflows
 -------------------------------------------------------------------------------
+`XGBoost <https://xgboost.readthedocs.io/en/latest/>`__ is an optimized distributed gradient boosting library designed to be efficient, flexible, and portable. 
+It uses `gradient boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`__ technique to implement Machine Learning algorithms.
+
+In this tutorial, we will understand how to predict house prices in multiple regions using XGBoost, and dynamic workflows in Flyte.
 
-In this tutorial, we will understand how to predict house prices in multiple regions using XGBoost, and :ref:`dynamic workflows <sphx_glr_auto_core_control_flow_dynamics.py>` in Flyte.
+A dynamic workflow is a typical workflow where the users can perform any arbitrary computation by consuming the inputs and producing the outputs. In the backend, it is designed as a task. During execution, the function body can be run to produce a workflow.
```

Comment:
Please revert the intro changes. Having a reference to dynamic workflow should be sufficient. We need not explain what it is again.